rotkehlchen/chain/substrate/manager.py

```diff
@@ -155,13 +270,12 @@ def get_accounts_balance(
             self,
             accounts: List[KusamaAddress],
     ) -> Dict[KusamaAddress, FVal]:
-        log.debug(
-            f'Querying {self.chain} chain for {self.chain_data.asset.identifier} balance',
-            account=accounts,
-        )
+        """
+        TODO error handling and return defaults
```

Comment:
Pending to address the error handling (NB: calling a decorated function) here and make it behave as expected when it fails for an account.